types/match.go

```diff
@@ -80,6 +81,11 @@ func (m *Matcher) Match(lset model.LabelSet) bool {
 	return string(v) == m.Value
 }
 
+// Equal checks whether two matchers are equal.
+func (m *Matcher) Equal(o *Matcher) bool {
```

Comment:
IIUC `Matchers.Equal()` isn't used anywhere in the Alertmanager repository so I would remove that method instead.

Reply:
how about change to private method? compare two arrays sometimes is useful.

Reply:
ok